### PR TITLE
Update Memberful WP profile page

### PR DESCRIPTION
**Pull request by mrhead**
This PR updates the plugin's profile page on wordpress.org. The PR updates files in the `assets/` directory; however, the release script currently syncs only changes from the plugin directory `wordpress/wp-content/plugins/memberful-wp`. I'll work on the release script update in a separate pull request.

---
* 7384bef **Update plugin tags**
  
  https://3.basecamp.com/3293071/buckets/5610646/todos/7236173297

* 69ed611 **Update header images**
  
  https://3.basecamp.com/3293071/buckets/5610646/todos/7236173520

* ee66e06 **Update contributors**
  
  https://3.basecamp.com/3293071/buckets/5610646/todos/7236331964

* 27b1207 **Update screenshots**
  
  https://3.basecamp.com/3293071/buckets/5610646/todos/7236324475

* da45181 **Update Description and Installation in readme.txt**
  
  https://3.basecamp.com/3293071/buckets/5610646/todos/7236174056

* bd534e3 **Update icons**
  
  https://3.basecamp.com/3293071/buckets/5610646/todos/7236308005

### CODE REVIEW

File: wordpress/wp-content/plugins/memberful-wp/readme.txt [hunk 1]
```diff
@@ -11,21 +11,68 @@ Sell memberships and restrict access to content with WordPress and Memberful.
...
+The Memberful WordPress plugin makes it easy to integrate Memberful with your WordPress website and protect content—serverside—for members only.
+
+Install Memberful on a self-hosted WordPress site or a site hosted with WordPress.com— Creator plan or higher. If you are on their Creator plan (or higher), and don’t want to upgrade, you can use the WordPress.com guided transfer service to move your content over to a self-hosted WordPress website.
+
+## Install the Memberful WordPress plugin
+
+1. From your Memberful dashboard navigate to Website → Settings and click the link to activate the WordPress service.
+2. Activate the WordPress service
+3. Copy the WordPress registration key (ctrl/cmd + c).
+4. Copy registration key
```
Comment (lmansur) [inline wordpress/wp-content/plugins/memberful-wp/readme.txt:55]: Item 4 seems to be a duplicate of 3

File: wordpress/wp-content/plugins/memberful-wp/readme.txt [hunk 2]
```diff
@@ -11,21 +11,68 @@ Sell memberships and restrict access to content with WordPress and Memberful.
...
+- Enable automatic tax calculations based on where you and your members are located.
 
 == Installation ==
 
-1. Install Memberful WP via the WordPress.org plugin directory, or download the file and visit Plugins => Add New => Upload from your WordPress dashboard.
-2. Activate the WordPress service from your Memberful account.
-3. From the plugin screen in your WordPress dashboard paste in your registration key and click the "Connect to Memberful" button.
+The Memberful WordPress plugin makes it easy to integrate Memberful with your WordPress website and protect content—serverside—for members only.
+
+Install Memberful on a self-hosted WordPress site or a site hosted with WordPress.com— Creator plan or higher. If you are on their Creator plan (or higher), and don’t want to upgrade, you can use the WordPress.com guided transfer service to move your content over to a self-hosted WordPress website.
```
Comment (lmansur) [inline wordpress/wp-content/plugins/memberful-wp/readme.txt:48]:
```suggestion
Install Memberful on a self-hosted WordPress site or a site hosted with WordPress.com— Creator plan or higher. If you are not on their Creator plan (or higher), and don’t want to upgrade, you can use the WordPress.com guided transfer service to move your content over to a self-hosted WordPress website.
```

**Review by lmansur — APPROVED** [review]
Nice! Got just two minor corrections.

File: wordpress/wp-content/plugins/memberful-wp/readme.txt (hunk 2 above)
Comment (mrhead) [reply wordpress/wp-content/plugins/memberful-wp/readme.txt:48]: Thanks!

**Review by julianfssen — APPROVED** [review]
Thanks, @mrhead !

**Pull request merged**